<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>472</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>363</width>
    <height>472</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chicken Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/images/chicken.png</normaloff>:/images/chicken.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Profile_tab">
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QListWidget" name="lw_profile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>331</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <item>
       <property name="text">
        <string>Default</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Play Doom</string>
      </property>
      <property name="text">
       <string>SCOOTALOO!!!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_new">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Новый</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_WADs">
     <attribute name="title">
      <string>WAD's</string>
     </attribute>
     <widget class="QListWidget" name="lw_iwad">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_pwad">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>54</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>PWAD:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>54</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>IWAD:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь до IWAD:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_iwad">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_pwad">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь до PWAD:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Порт</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>21</y>
         <width>341</width>
         <height>141</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>----Singleplayer----</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GZDoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ZDoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chocolate Doom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PrBoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PrBoom-plus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LxDoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>----3D----</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EDGE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Risen3D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Doomsday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vavoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>----Multiplayer----</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ZDaemon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zandronum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odamex</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Дополнительные параметры порта:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь до исполняемого файла порта:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Дополнительно</string>
     </attribute>
     <widget class="QPushButton" name="btn_about">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
